src/components/ConfirmationModal.vue

```diff
@@ -67,11 +67,31 @@ export default {
 
 <style lang="scss" scoped>
 .confirm-modal {
-	padding: 20px;
+	padding: 24px 32px;
+
+	h2 {
+		margin-bottom: 20px;
+		font-size: 24px;
+		font-weight: 600;
+	}
```

Comment:
you dont need this part, the buttons have the styling from nc/vue library and it should stay that way, so it looks the same everywhere